stl/inc/mdspan

```diff
@@ -1026,9 +1027,13 @@ public:
     }
 
 private:
-    template <class _IndexContainer, size_t... _Idx>
-    _NODISCARD constexpr reference _Index_impl(_IndexContainer&& _Indices, index_sequence<_Idx...>) const {
-        return _Acc.access(_Ptr, _Map(_STD as_const(_STD forward<_IndexContainer>(_Indices)[_Idx])...));
+    template <class _OtherIndexType, size_t... _Seq>
+    _NODISCARD constexpr reference _Index_impl(span<_OtherIndexType, rank()> _Indices, index_sequence<_Seq...>) const {
+#ifdef __clang__ // TRANSITION, P2128R6
+        return this->operator[](_STD as_const(_Indices[_Seq])...);

```

Comment:
Do we need `this->` here? It appears to be a direct member, so I think this is unnecessary.